core/orchestrator_core.go

```diff
@@ -54,6 +54,21 @@ func recordTiming(operation string, err *error) func() {
 	}
 }
 
+// recordTransactionTiming records the duration of a transaction in milliseconds
+// and whether it was successful or not.
+// Parameters:
+//   txn: the transaction to record
+//   err: the error that occurred, or nil if no error occurred
+// Returns:
+//   nothing
+// Example:
+//   err := c.transaction(func(txn *storage.VolumeTransaction) error {
+//     // do something
+//     return nil
+//   })
+//   recordTransactionTiming(txn, &err)
+//
+// -- Doc autogenerated on 2022-05-26 22:14:22. Function hash: d39f7d63300a6cea90e7402ac7280126 --
```

Comment:
The date is nice, but not sure we need the timestamp too.  We have 78 different timestamps in the file:
```
% grep "autogenerated on" orchestrator_core.go | wc -l
78
```

Reply:
Can you expand on that? This is for human consumption right? Or do you have something else in mind?

Reply:
I'll remove the hour::minute:second string